src/core/android/SDL_android.c

```diff
@@ -2821,4 +2824,37 @@ bool Android_JNI_OpenFileDialog(
     return true;
 }
 
+SDL_FormFactor SDL_GetAndroidDeviceFormFactor(void)
+{
+    JNIEnv *env = Android_JNI_GetEnv();
+    SDL_FormFactor form_factor = SDL_FORMFACTOR_UNKNOWN;
+    jstring string;
+
+    string = (*env)->CallStaticObjectMethod(env, mActivityClass, midClipboardGetText);
```

Comment:
Should this be midGetDeviceFormFactor instead of midClipboardGetText?

Reply:
Indeed, thanks for the catch!